### ISSUE TITLE
Too much `any` to be useful

### ISSUE THREAD

**Issue by Klustre** [adobe/cc-ext-uxp-types#9]
### Expected Behaviour

TypeScript

### Actual Behaviour

JavaScript

#### Sample Code that illustrates the problem

<img width="591" alt="any" src="https://github.com/user-attachments/assets/c708a04c-eb36-46a5-9494-27fc38f95778">

**Comment by justintaylor-dev** [comment]
Agreed, Auto-Generated types need to be corrected upstream for these to be correct.

There's no point in having type defs if 1000+ are set to `any`.

Thanks
